wp-content/themes/flightfindertheme/functions.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Twenty Fifteen functions and definitions
+ */
+ 
+/* Scripts should be inserted in <head> */
+function head_scripts() {
+	echo '<script src="//code.jquery.com/jquery-2.1.4.min.js"></script>
```

Comment:
This does not work, do not use the absolute path. Use https://developer.wordpress.org/reference/functions/wp_enqueue_script/